tests/acceptance/features/webUISharingInternalUsers/shareWithUsers.feature

```diff
@@ -1,11 +1,13 @@
-@skipOnOCIS @ocis-reva-issue-64
```

Comment:
@skipOnOCIS @ocis-reva-issue-64

Reply:
@dpakach it's supposed to run on ocis, `...ToRoot/shareWithUsers.feature` is skipped on ocis.

Reply:
You should still skip these because these tests still don't work on ocis